dbms/src/Flash/Mpp/BroadcastOrPassThroughWriter.cpp

```diff
@@ -75,7 +75,19 @@ void BroadcastOrPassThroughWriter<ExchangeWriterPtr>::encodeThenWriteBlocks()
     }
     assert(blocks.empty());
     rows_in_blocks = 0;
+    auto packet_bytes = tracked_packet->getPacket().ByteSizeLong();
     writer->broadcastOrPassThroughWrite(std::move(tracked_packet));
+
+    {
+        auto tunnel_cnt = writer->getPartitionNum();
+        size_t local_tunnel_cnt = 0;
+        for (size_t i = 0; i < tunnel_cnt; ++i)
+        {
+            local_tunnel_cnt += writer->isLocal(i);
+        }
+        GET_METRIC(tiflash_exchange_data_bytes, type_broadcast_passthrough_original_all).Increment(packet_bytes * tunnel_cnt);
+        GET_METRIC(tiflash_exchange_data_bytes, type_broadcast_passthrough_none_local).Increment(packet_bytes * local_tunnel_cnt);
```

Comment:
broadcast write does not enable compression?

Reply:
No. Currently the data used for broadcast write are relative small. We can support it later if necessary.